starknet_py/net/client_models.py

```diff
@@ -332,7 +332,16 @@ class ContractClass:
 
     program: dict
     entry_points_by_type: EntryPointsByType
-    abi: Optional[AbiDictList] = None
+    abi: Optional[AbiDictList]
```

Comment:
Shouldn't we leave abi default to None?

Reply:
It cannot be `= None` in the current implementation, as **CompiledContract** inherits from **ContractClass** and it has would cause "non default argument after a default argument"

Reply:
Right, forget about it